core/templates/dev/head/editor/exploration_history.html

```diff
@@ -20,12 +20,10 @@
   <md-card class="oppia-editor-card">
     <h3>List of Changes</h3>
     <div ng-repeat="versionCheckbox in versionCheckboxArray " class="row">
-      <div class="col-sm-2 col-md-2 col-lg-2">
+      <div class="col-sm-2 col-md-4 col-lg-4">
```

Comment:
I think, col-sm-4 will be better. See the differences on a bit smaller screen,
**Using col-sm-2**
![screenshot from 2016-05-15 18 48 34](https://cloud.githubusercontent.com/assets/11073943/15274452/77f6bb5e-1acf-11e6-8245-85f14e248c2d.png)
**Using col-sm-4**
![screenshot from 2016-05-15 18 48 38](https://cloud.githubusercontent.com/assets/11073943/15274454/7dfb8dfe-1acf-11e6-80cc-863551ad436c.png)